intentkit/clients/cdp.py

```diff
@@ -32,22 +32,29 @@ async def get_wallet_provider(self) -> CdpEvmServerWalletProvider:
 
         logger = logging.getLogger(__name__)
 
-        # Collect credentials from system-config first, then env vars as fallback
+        # Get credentials from config (which handles AWS secrets and env vars)
         api_key_id = (
             self._skill_store.get_system_config("cdp_api_key_name")
-            or os.getenv("CDP_API_KEY_ID")
-            or os.getenv("CDP_API_KEY_NAME")
+            or config.cdp_api_key_id
```

Comment:
don't use "or" to fallback,  self._skill_store.get_system_config("cdp_api_key_name") is enough, More places, more chaos.